src/taskpane/components/InputField.tsx

```diff
@@ -0,0 +1,19 @@
+import { TextField } from "@fluentui/react";
+import { useField } from "formik";
+
+import * as React from "react";
+
+type InputFieldProps = React.InputHTMLAttributes<HTMLInputElement> & {
+  label: string;
+  name: string;
+};
+
+// eslint-disable-next-line react/prop-types
+export const InputField: React.FC<InputFieldProps> = ({ defaultValue, ...props }) => {
```

Comment:
What are your thoughts on

```suggestion
export const InputField: React.FC<InputFieldProps> = ({ defaultValue, ...props }: InputFieldProps) => {
```

?

It is similar to [these examples](https://github.com/typescript-cheatsheets/react#function-components) in the TS react cheat sheet.

Reply:
Oups, saw that you already used that in the wrapper, ignore me X)